src/components/App/App.js

```diff
@@ -4,38 +4,54 @@ import treeData from '../../assets/treeData';
 import TreeDisplay from '../TreeDisplay/TreeDisplay';
 import Header from '../Header/Header';
 import Form from '../Form/Form';
+import { Route } from 'react-router-dom';
 
 class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      trees: []
+      trees: [],
+      currentTree: {}
     }
   }
 
   componentDidMount() {
-    this.setState({ trees: treeData })
+    this.setState({ 
+      trees: treeData,
+      currentTree: treeData[this.getRandomIndex(0, treeData.length - 1)] 
```

Comment:
What a clever way to get this setState situation resolved! and glad we were able to rework the setNewTree method to just get a random tree from this.state.trees so we aren't doing network requests every time we click that button.